mfthomps/CyberCIEGE-port-to-Unity3d
# Merge Request for Issue/298 - groups/clearances view

Closes #298

To test:
- Run Starting Scenarios/TirePly Filter Scenario
- Click Clearance tab to open Clearance View
- Click on clearance to view its description and edit its background check level
- Repeat with Group tab and View
- Make sure users' highest background check level matches the highest background check level of their clearance and group
